userapi/src/main/java/fk/prof/userapi/util/ProtoUtil.java

```diff
@@ -0,0 +1,46 @@
+package fk.prof.userapi.util;
+
+import com.google.protobuf.*;
+import fk.prof.aggregation.proto.AggregatedProfileModel;
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.Unpooled;
+import io.vertx.core.buffer.Buffer;
+import recording.Recorder;
+
+import java.io.IOException;
+
+public class ProtoUtil {
```

Comment:
Is this copied. Add a //TODO

Reply:
Added a TODO [here](https://github.com/Flipkart/fk-prof/commit/d05ec11fd677af720091abf5af0769329510189b)